src/player.cpp

```diff
@@ -438,6 +438,11 @@ void Player::updateInventoryWeight()
 			inventoryWeight += item->getWeight();
 		}
 	}
+
+	StoreInbox* storeInbox = getStoreInbox();
+	if (storeInbox) {
```

Comment:
if (StoreInbox* storeInbox = getStoreInbox()) {

Reply:
of course, how could I forget to compact this into one line ;)